src/screens/app/Components/BookmarkButton/BookmarkButton.js

```diff
@@ -1,6 +1,5 @@
 import {IconButton} from 'react-native-paper';
 import React from 'react';
```

Comment:
Ensure this file doesn't conflict with what's on master. Otherwise remove commit.